src/assets/translations/en/main.json

```diff
@@ -407,6 +407,8 @@
     "noSupportedAssets": "No supported assets in portfolio."
   },
   "trade": {
+    "allowance": "Allowance",
+    "allowanceTooltip": "Lorem ipsum sit dolor amet",
```

Comment:
We are waiting on @shapeshift/product for this one, yeah?

Reply:
Yep, asked it on the issue but we are waiting for a response